integrationtests/tests/fixtures/mod.rs

```diff
@@ -822,6 +880,31 @@ impl FederationTest {
         user.client.fetch_all_notes().await.unwrap();
     }
 
+    /// Verifies that the database migration for the dummy module completed successfully
+    pub async fn verify_dummy_module_database_migration(&self, expected_example_key_size: usize) {
+        for server in &self.servers {
+            block_on(async {
```

Comment:
AFAIR, these `block_on` calls are a bad workaround for warning about holding `borrow`/`borrow_mut` of `RefCell` across `yield`? They are not making the problem go away, just mask it. It would be more honest to just use `#[allow(...)]` with a comment.

After https://github.com/fedimint/fedimint/pull/1599 the need for them should no longer be there.

BTW. We should never need to use `Rc<RefCell<_>>` that a type that should (almost) never appear in a multi-threaded runtime async code.

Reply:
Yes this was just done to appease the compiler :) 

Should I just wait until https://github.com/fedimint/fedimint/pull/1599 lands then fix it here?

Reply:
Don't wait, but after #1599 would be nice if you could clean it up here and whenever else (should be just `grep` away).

Reply:
Changed to `#[allow(...)]` for now, can remove in follow up

Reply:
#1599 is merged

Reply:
Removed `#[allow(...)]`